wscript

```diff
@@ -421,6 +421,11 @@ audio_output_features = [
         'desc': 'SDL2 audio output',
         'deps': 'sdl2',
         'func': check_true,
+    }, {
+        'name': '--oss-audio',
+        'desc': 'OSS',
+        'func': check_statement(['sys/soundcard.h'], 'int x = AFMT_S16_NE'),
```

Comment:
Linux [has `AFMT_S16_NE`](https://github.com/torvalds/linux/search?q=AFMT_S16_NE), so OSS would be enabled by default. Maybe check [`SNDCTL_DSP_SETPLAYVOL`](http://manuals.opensound.com/developer/SNDCTL_DSP_SETPLAYVOL.html) instead.